README.rst

```diff
@@ -568,6 +569,11 @@ Variables can also be defined in `Device groups <#device-groups>`__.
 Refer the `Group configuration variables <group-configuration-variables>`_
 section for detailed information.
 
+Organization variables
+~~~~~~~~~~~~~~~~~~~~~~
+
+Variables can also be defined at the organization level.
```

Comment:
Please maintain consistency with "Group Variables" by adding a similar brief section about it and linking to it.

Reply:
The [Group Configuration Variables](https://github.com/openwisp/openwisp-controller#group-configuration-variables) section is under the [Device Group](https://github.com/openwisp/openwisp-controller#device-groups) section.

Reply:
I have updated the documentation, please check https://github.com/openwisp/openwisp-controller/tree/issues/780-organization-variables#organization-variables